File: BasePlugin/src/main/java/com/kronos/plugin/base/Log.java
package com.kronos.plugin.base;

import org.codehaus.groovy.runtime.InvokerHelper;

class Log {

    public static void info(Object msg) {
        try {
            System.out.println(InvokerHelper.
                    toString(String.format("{%s}", msg.toString())));
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: BasePlugin/src/main/java/com/kronos/plugin/base/Log.kt
package com.kronos.plugin.base

import org.codehaus.groovy.runtime.InvokerHelper

object Log {
    @JvmStatic
    fun info(msg: Any) {
        try {
            println(InvokerHelper.toString(String.format("{%s}", msg.toString())))
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}